pkg/distributor/distributor_test.go

```diff
@@ -439,7 +507,7 @@ func TestDistributor_PushIngestionRateLimiter(t *testing.T) {
 				{metadata: 1, expectedError: httpgrpc.Errorf(http.StatusTooManyRequests, "ingestion rate limit (5) exceeded while adding 0 samples and 1 metadata")},
 			},
 		},
-		"burst should set to each distributor": {
+		"ingestion burst should set to each distributor": {
```

Comment:
```suggestion
		"For each distributor, set an ingestion burst limit.": {
```
I think you are referring to some sort of limit. Also, vet this for technical accuracy. I reworded to remove the ambiguous term _should_ and am not sure if my suggestion is accurate.